connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewClient.kt

```diff
@@ -41,13 +56,14 @@ internal class StripeConnectWebViewClient(
                 userAgentString = "$userAgentString - stripe-android/${StripeSdkVersion.VERSION_NAME}"
             }
             addJavascriptInterface(StripeJsInterface(), ANDROID_JS_INTERFACE)
-            addJavascriptInterface(StripeJsInterfaceInternal(), ANDROID_JS_INTERNAL_INTERFACE)
+            addJavascriptInterface(StripeJsInterfaceInternal(this), ANDROID_JS_INTERNAL_INTERFACE)
 
             loadUrl(embeddedComponentManager.getStripeURL(connectComponent))
         }
     }
 
     private fun initJavascriptBridge(webView: WebView) {
+        logger.debug("Initializing Javascript Bridge")
         webView.evaluateJavascript(
```

Comment:
\nit we can move these to `evaluateSdkJs`
```suggestion
        webView.evaluateSdkJs(
```